apps/examples/mediarecorder/mediarecorder_main.cpp

```diff
@@ -37,10 +37,11 @@ constexpr int RECORDER_START = 2;
 constexpr int RECORDER_PAUSE = 3;
 constexpr int RECORDER_RESUME = 4;
 constexpr int RECORDER_STOP = 5;
-constexpr int VOLUME_UP = 6;
-constexpr int VOLUME_DOWN = 7;
-constexpr int PLAY_DATA = 8;
-constexpr int DELETE_FILE = 9;
+constexpr int GET_MAX_VOLUME = 6;
```

Comment:
Is there a particular reason for using constexpr instead of enum?
As I can see, since these values are used purely in the form of constants, we dont need any typechecking and so the use of enums might be more clean method, instead of making them individual variables which would take up space in the data section.

Reply:
Ok, Modified.